pages/common/batch.md

```diff
@@ -0,0 +1,16 @@
+# batch
+
+> Execute commands at a later time when the system load levels permit.
+> \*Make sure the atd (or atrun) is running.
```

Comment:
Removed the `*` for now. Let me know what is the best way to differentiate this text from man description.